app/javascript/forms/mappers/validatorMapper.jsx

```diff
@@ -13,10 +13,10 @@ const fileValidator = ({ maxSize }) => {
     const imageTypes = /image\/jpg|image\/jpeg|image\/png|image\/svg/;
 
     if (value && value.inputFiles[0] && !imageTypes.test(value.inputFiles[0].type))
-      return __(`File must be an image of type "png", "jpg/jpeg", or "svg". The currently uploaded file's extension is "${value.inputFiles[0].type.split('/').pop()}"`);
+      return sprintf(__(`File must be an image of type "png", "jpg/jpeg", or "svg". The currently uploaded file's extension is "%s"`), value.inputFiles[0].type.split('/').pop());
 
     if (value && value.inputFiles[0] && value.inputFiles[0].size > maxSize)
-      return __(`File is too large, maximum allowed size is ${maxSize} bytes. Current file has ${value.inputFiles[0].size} bytes`);
+      return sprintf(__(`File is too large, maximum allowed size is %s bytes. Current file has %s bytes`), maxSize, value.inputFiles[0].size);
```

Comment:
`bytes` should probably follow localization rules for pluralization - not sure how to do that in javascript offhand.

Reply:
not sure what you mean, for this alert the `if (value && value.inputFiles[0] && value.inputFiles[0].size > maxSize)` statement forces the size to be greater than the `maxSize`, so i feel value.inputFiles[0].size will always be plural. Do you think there will be a case where the max size is `1 byte`? 

```
      return sprintf(
        n__(
          `File is too large, maximum allowed size is %s byte. Current file has %s bytes`,
          `File is too large, maximum allowed size is %s bytes. Current file has %s bytes`, 
          value.inputFiles[0].size
        ), 
        maxSize, value.inputFiles[0].size);

Reply:
Ah ok, that's fair 👍